internal/pkg/version/version.go

```diff
@@ -4,5 +4,19 @@
 // Package version holds variables for generating version information
 package version
 
+const (
+	// LegacyAppTemplateVersion is the version associated with the application template before we started versioning.
+	LegacyAppTemplateVersion = "v0.0.0"
+	// AppTemplateMinVersionAlias is the least version number available for HTTPS alias.
+	AppTemplateMinVersionAlias = "v1.0.0"
+	// AppTemplateMinVersionStaticSite is the minimum app version required to deploy a static site.
+	AppTemplateMinVersionStaticSite = "v1.2.0"
+)
+
 // Version is this binary's version. Set with linker flags when building Copilot.
 var Version string
+
+// LatestTemplateVersion is the latest version number available for Copilot templates.
+func LatestTemplateVersion() string {
```

Comment:
why do we need this? isn't it the same as `version.Version`?

Reply:
`version.Version` is a variable that can be altered by mistake. Previously we used const so it wasn't a problem before.